torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1230,8 +1230,8 @@ at::Tensor XLANativeFunctions::clone(
     const at::Tensor& self,
     std::optional<at::MemoryFormat> /* memory_format */) {
   TORCH_LAZY_FN_COUNTER_TIMED_TRACING("xla::");
-  return bridge::AtenFromXlaTensor(
-      tensor_methods::clone(bridge::GetXlaTensor(self)));
+  return bridge::AtenFromXlaTensor(tensor_methods::clone(
+      bridge::GetXlaTensor(self), /* is_conj= */ self.is_conj()));
```

Comment:
What does "is_conj" mean? If I call conj on a tensor, I would expect that to compute the complex conjugate. Why does a tensor need to know if someone did a conj op on it?

Reply:
Got it